backend/services/mining/src/MiningPool.py

```diff
@@ -0,0 +1,57 @@
+import threading
+import time
+
+class MiningPool:
+    DEQUEUE_TIME = 5
+
+    def __init__ (self, receiver, ready_to_mine):
+        self._pool = []
+        self._lock = threading.Lock()
+        self._mining_thread = threading.Thread(target=self.sendToMine, kwargs={'self': self, 'receiver': receiver})
```

Comment:
All thread/lock stuff can be removed as per our conversation. I'd hazard a guess that clients and the service can push/pop to the existing list and things will be fine. However, to ensure this, we can use a Queue instead of a list. Either way, threading is unnecessary and can be removed to reduce complexity